src/app/content/utils/seoUtils.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Book, Page } from '../types';
+import { makeArchiveSection, makeArchiveTree } from './archiveTreeUtils.spec';
```

Comment:
Pretty sure that importing an another test file will make it impossible to run `seoUtils` alone ( and i'm not sure if it's a good idea to make test files depend on each other). I'm not sure where to move the utils but i dont think this is a good idea. Maybe @TomWoodward could help.

Reply:
running `seoUtils` alone works but I agree that tests (or actually any files) should not depend on each other like this. I wasn't sure if `test/utils.ts` is a proper place for this.
I could also just duplicate it or create a mock `page` and `chapter` instead of using a function

Reply:
these are mocks right? you can move them to `src/test/mocks/`